acados/utils/print.h

```diff
@@ -75,8 +75,12 @@ void print_ocp_qp_dims(ocp_qp_dims *dims);
 
 void print_ocp_qp_in(ocp_qp_in *qp_in);
```

Comment:
In general all printing routines that are printing HPIPM structures I would move them in HPIPM.
Maybe it's unrelated to this PR but it would make sense to do at some point.

Reply:
I decided to just leave a note, stating that these routines should be moved to HPIPM, for now.